### DISCUSSION TITLE
Question regarding blog

### DISCUSSION THREAD

**Discussion by Swepool** [sveltekit-i18n/lib#68] [Q&A] [answered]
How could I handle locale routing with a markdown blog? Can't figure it out 🤔

**Comment by jarda-svoboda** [comment]
Hi @Swepool!
Thanks for a great question!.) Currently, there is no support for automatic translations, however you could use MDsveX which should work normally with translation placeholders (but not tested so far - example will be created in near future). 

Connectors (for Google Translations API etc.) are planned in version 3..

You can also find similar question [here](https://github.com/sveltekit-i18n/lib/discussions/67).

[a comment by an automated account is not shown]
